Documentation/release-notes/README.md

```diff
@@ -0,0 +1,177 @@
+itself is largely irrelevant; the point is that it be sufficiently unique so as
+to avoid potential merge conflicts with other PRs.
+
+## Templates
+
+### Build time or app performance improvement
+
+  * Use a command form verb in the first sentence, as if you were telling
+    someone to do something.
+  * Include a before and after time comparsion if possible.
+
+Examples:
+
+```
+  * [GitHub PR 3640](https://github.com/xamarin/xamarin-android/pull/3640):
+    Use System.Reflection.Metadata rather than Cecil for
+    `ResolveLibraryProjectImports`.  This reduced the time for the
+    `ResolveLibraryProjectImports` task from about 4.8 seconds to about 4.5
+    seconds for a small test Xamarin.Forms app on an initial clean build.
+
+  * [GitHub PR 3729](https://github.com/xamarin/xamarin-android/pull/3729):
+    Initialize application logging and uncaught exception handling lazily.  This
+    reduced the time to display the first screen of a small test Xamarin.Forms
+    app from about 783 milliseconds to about 754 milliseconds for a Release
+    configuration build on a Google Pixel 3 XL device.
+```
+
+### New feature or behavior
+
+  * Describe what the change is, why it was added, and how to enable it or
+    account for it.
+
+Example:
+
+````
+#### XA0119 warning for non-ideal Debug build configurations
+
+Xamarin.Android 10.1 adds a new XA0119 build warning to identify cases where
+projects might have unintentionally enabled settings that slow down deployments
+in the Debug configuration.  For the best Debug configuration deployment
+performance, first ensure that **Use Shared Runtime** is enabled in the Visual
+Studio project property pages.  This sets the `$(AndroidUseSharedRuntime)`
+MSBuild property to `true` in your *.csproj* file:
+
+```xml
+<PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+  <AndroidUseSharedRuntime>true</AndroidUseSharedRuntime>
+</PropertyGroup>
+```
+
+When **Use Shared Runtime** is enabled, an XA0119 warning will appear if any of
+the following settings are enabled:
+
+  * **Android Package Format** set to **bundle**
+  * **Code shrinker**
+  * **AOT Compilation**
+  * **Enable Startup Tracing**
+  * **Linking**
+
+Be sure to disable these settings for the best Debug configuration deployment
+performance.
+````
+
+### External tool or library version update
+
+Example:
+
+```
+#### Mono.Data.Sqlite SQLite version update
+
+The version of SQLite used by Mono.Data.Sqlite in Xamarin.Android has been
+updated from 3.27.1 to [3.28.0](https://sqlite.org/releaselog/3_28_0.html),
+bringing in several improvements and bug fixes.
+```
+
+### Other issue fix or small improvement
+
+  * Use past tense verbs when describing the previous problematic behavior.
+  * List all public GitHub or Developer Community items that were fixed by the
+    change.  If no public items were fixed, or if the scope of the fix was
+    larger than any of the individual items, list the PR instead.
```

Comment:
On second thought, this probably needs to be something like:

```diff
-larger than any of the individual items, list the PR instead.
+larger than any of the individual items, skip adding any items, or if you
+already have a PR number, list that instead.
```

There won't be a PR number yet in cases where the draft release note is added before the PR is even started.